tidynamics/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "tidynamics" %}
+{% set version = "0.1.2.dev4" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "7b787ff9392c19194b483852a065cbdb4463a2f78958833b2b42a160d9730c6d" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  noarch: python
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - numpy
+  run:
+    - python
+    - numpy
+
+test:
+  imports:
+    - tidynamics
+
+about:
+  home: ' https://pypi.python.org/pypi/tidynamics'
+  license: 3-clause BSD
+  license_family: BSD
+  license_file: LICENSE
```

Comment:
You might consider including the `license_file` in the source distribution on pypi by adding the appropriate `MANIFEST.in` file so that the license doesn't need to be packaged manually here. This isn't necessary to get this recipe merged, but for future updates to the feedstock.